ote/src/cljs/ote/style/base.cljs

```diff
@@ -207,7 +207,7 @@
    :align-items "center"})
 
 (def icon-labeled-icon
-  {:margin-right "0.25rem"})
+  {:margin-right "0.125rem"})
```

Comment:
Just double checking, was it checked that all places where this affects look good? 
e.g. success_msg and warning_msg.

Reply:
Change is small and I think that it won't make success messages to look bad.
But I didn't test it. 
But it looks good in transit visualization page even when icons has trailing text (<icon> Mahollisesti päättyvä reitti)